frontend/src/components/dashboard/subdomain/ConfirmationForm.module.scss

```diff
@@ -40,21 +40,34 @@
     justify-content: space-between;
     padding-top: $spacing-xl;
 
+    button[type="submit"] {
+      background: $color-blue-50;
+      opacity: 0.4;
```

Comment:
Might be more future-proof if we only set this opacity on disabled buttons, rather than first setting it and then removing it if it's _not_ disabled?

Reply:
Btw, might we not want this disabled styling for all buttons, rather than just the one in this dialog? i.e. shouldn't these changes be in `Button.module.scss`?

Reply:
Good point! I'll add this into `Button.module.scss` instead.